docs/schema-v2.old.md

```diff
@@ -0,0 +1,160 @@
+# Skybox specification
```

Comment:
Considering I never released anything with this schema version, there's no need to have a old version

Reply:
I’ve just kept that for my reference. 
was planning to clean up after I finish the docs